src/engine/DrawManager.java

```diff
@@ -2641,20 +2641,37 @@ public void drawGhost_2p(boolean levelFinished, double lives_1p, double lives_2p
 			System.out.println(lives_1p);
 			System.out.println(lives_2p);
```

Comment:
[P2]

It looks like the code was written for testing purposes, 
but I think it would be better to use logger.